index.html

```diff
@@ -11,6 +11,49 @@
 </head>
     <body>
 
+        <div id="book1">
+            <h1>Name</h1>
```

Comment:
I like the labels, but there needs to be a field for our eventual content as well